src/status_im/ui/screens/communities/communities_list_redesign.cljs

```diff
@@ -206,34 +207,34 @@
   (let [multiaccount (<sub [:multiaccount])
         communities (<sub [:communities/communities])
         featured-communities (<sub [:communities/featured-communities])]
-    (fn []
-      [safe-area/consumer
-       (fn [insets]
-         [react/view {:style {:flex             1
-                              :padding-top      (:top insets)
-                              :background-color (quo2.colors/theme-colors
-                                                 quo2.colors/neutral-5
-                                                 quo2.colors/neutral-95)}}
-          [topbar/topbar
-           {:navigation      :none
-            :left-component  [react/view {:margin-left 16}
-                              [photos/photo (multiaccounts/displayed-photo multiaccount)
-                               {:size 32}]]
-            :right-component [react/view {:flex-direction :row
-                                          :margin-right 16}
-                              [topnav/qr-scanner]
-                              [topnav/qr-code]
-                              [topnav/notifications-button]]
-            :new-ui?         true
-            :border-bottom   false}]
-          [title-column]
-          [react/scroll-view
-           [community-filter-tags]
-           [featured-communities-section featured-communities]
-           (when communities
-             [:<>
-              [react/view {:margin-vertical    4
-                           :padding-horizontal 20}
-               [separator/separator]]
-              [community-segments]])
-           [community-segments-view communities]]])])))
```

Comment:
Do we need this change?

Reply:
@ilmotta requested this in the previous p.r. right Icaro?